server/middleware/barcode/barcodeAuthorisationMiddleware.ts

```diff
@@ -1,36 +1,13 @@
 import { RequestHandler } from 'express'
-import type { VerifyErrors, JwtPayload } from 'jsonwebtoken'
-import { verify } from 'jsonwebtoken'
-import config from '../../config'
 
 export default function barcodeAuthorisationMiddleware(): RequestHandler {
   return (req, res, next) => {
     res.locals.barcodeUser = undefined
 
-    if (!req.session.slmToken) {
+    if (!req.session.slmToken || !req.session.validSlmToken) {
       return res.redirect('/link/request-link')
     }
 
-    const token = req.session.slmToken
-    req.session.validSlmToken = false
-    return verify(
```

Comment:
The CJSM auth cookie is not rolling so doesn't need to be updated every request. Hence it has been moved to the VerifyLinkController.